### PR TITLE
Remove FF_SPIKE_SMS_DAILY_LIMIT and FF_SMS_PARTS_UI

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Remove  
FF_SPIKE_SMS_DAILY_LIMIT
FF_SMS_PARTS_UI

and related tests

#Testing:

Please go through the frontend and see we are not missing anything (eyeball)

### CODE REVIEW

File: app/templates/views/notifications/check.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
   {% endif %}
 
   {{ template|string|translate_preview_template }}
-  {% if config["FF_SMS_PARTS_UI"] and template.template_type == 'sms'%}
+  {% if template.template_type == 'sms'%}
```
Comment (amazingphilippe) [inline app/templates/views/notifications/check.html:78]: Adds content about "message parts"
image

**Review by andrewleith — COMMENTED** [review]
Tests and code changes look good.  The app behaves as expected.

Once you remove the `Usage today` section I'll re-test and sign-off.

File: Makefile [hunk 2]
```diff
@@ -59,6 +59,7 @@ run-dev:
 format:
 	isort ./app ./tests
 	black ./app ./tests
+	flake8 ./app ./tests
```
[the remark this answers is not in the document]
Comment (jzbahrai) [reply Makefile:62]: unsure

**Review by andrewleith — APPROVED** [review]
LGTM!

**Pull request merged**